web/src/lightbox.js

```diff
@@ -177,18 +177,32 @@ export function clear_for_testing() {
 export function render_lightbox_list_images(preview_source) {
     if (!is_open) {
         const images = Array.prototype.slice.call(
-            $(".focused-message-list .message_inline_image img"),
+            $(
+                ".focused-message-list .message_inline_image img, .focused-message-list .message_inline_video video",
+            ),
         );
         const $image_list = $("#lightbox_overlay .image-list").empty();
 
         for (const img of images) {
             const src = img.getAttribute("src");
             const className = preview_source === src ? "image selected" : "image";
+            const is_video = img.tagName === "VIDEO";
 
-            const $node = $("<div>")
-                .addClass(className)
-                .attr("data-src", src)
-                .css({backgroundImage: "url(" + src + ")"});
+            let $node;
+            if (is_video) {
+                $node = $("<div>").addClass(className).addClass("video").attr("data-src", src);
```

Comment:
Should we call that class `lightbox_video`? The bare `video` class name seems like it could match something used elsewhere in the app by accident.

Reply:
Done